operations/CSM_product_management/Configure_Non-Compute_Nodes_with_CFS.md

```diff
@@ -73,8 +73,8 @@ keys.
     > key files on the system.
 
     ```bash
-    ncn# PUBLIC_KEY_FILE=/root/.ssh/id_rsa-csm.pub
-    ncn# PRIVATE_KEY_FILE=/root/.ssh/id_rsa-csm
+    ncn# PUBLIC_KEY_FILE=/path/to/id_rsa-csm.pub
```

Comment:
I'd rather see this use $HOME instead.

Reply:
The reason I changed it from that is because this is the procedure for using custom SSH keys, instead of using the keys for the root user. There is a different procedure to follow to use the user's normal keys. Or put another way, this isn't a case where they would be copying and pasting -- they'd need to point to the custom SSH keys they want to use.